File: frontend/src/app/Business/Models/veiculo1.ts
import { RomaneioCarga } from "./romaneioCarga";
import { Tenant } from "./tenant";
import { Veiculo } from "./veiculo";

export interface Veiculo1  {
    id: string;
    tenantId: string | null;
    veiculoId: string | null;
    dataCadastro: string | null;
    romaneioCargas: RomaneioCarga[];
    tenant: Tenant | null;
    veiculo: Veiculo | null;
}

File: backend - Copia/Entities/Models/Veiculo1.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class Veiculo1
{
    public Guid Id { get; set; }

    public Guid? TenantId { get; set; }

    public Guid? VeiculoId { get; set; }

    public Guid? MotoristaId { get; set; }

    public DateTime? DataCadastro { get; set; }

    public virtual Motoristum? Motorista { get; set; }

    public virtual ICollection<RomaneioCarga> RomaneioCargas { get; } = new List<RomaneioCarga>();

    public virtual Tenant? Tenant { get; set; }

    public virtual Veiculo? Veiculo { get; set; }
}
